tests/parsers/gdrive_synclog.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""Tests for the GoogleDriveSyncLog log parser."""
+
+from __future__ import unicode_literals
+
+import unittest
+
+from plaso.formatters import gdrive_synclog as _  # pylint: disable=unused-import
+from plaso.lib import timelib
+from plaso.parsers import gdrive_synclog
+
+from tests import test_lib as shared_test_lib
+from tests.parsers import test_lib
+
+# TODO: more tests, esp. for confirming Unicode support, other OS discrepancies.
```

Comment:
Please elaborate what you specifically need to additionally test. Maybe add an github issue to track progress.

Reply:
Removed the TODO, since I think the test case as-is is OK, but it would be nice to generate more sync_log.log files (with a system in UTC, on another OS, or filenames containing Unicode characters) and test those. I can create a Github issue for that after this.